packages/mui-material/src/ListItem/ListItem.d.ts

```diff
@@ -151,35 +132,8 @@ export interface ListItemTypeMap<AdditionalProps, RootComponent extends React.El
  *
  * - [ListItem API](https://mui.com/material-ui/api/list-item/)
  */
-declare const ListItem: ExtendButtonBase<
-  ListItemTypeMap<
-    {
-      /**
-       * If `true`, the list item is a button (using `ButtonBase`). Props intended
-       * for `ButtonBase` can then be applied to `ListItem`.
-       * @default false
-       * @deprecated checkout [ListItemButton](/material-ui/api/list-item-button/) instead
-       *
-       */
-      button: true;
-    },
-    'div'
-  >
-> &
-  OverridableComponent<
-    ListItemTypeMap<
-      {
-        /**
-         * If `true`, the list item is a button (using `ButtonBase`). Props intended
-         * for `ButtonBase` can then be applied to `ListItem`.
-         * @default false
-         * @deprecated checkout [ListItemButton](/material-ui/api/list-item-button/) instead
-         */
-        button?: false;
-      },
-      'li'
-    >
-  >;
+declare const ListItem: ExtendButtonBase<ListItemTypeMap<{}, 'div'>> &
```

Comment:
We should remove `ExtendButtonBase<ListItemTypeMap<{}, 'div'>> &` as ListItem no longes has a dependency on it.

Reply:
Oh, that makes sense! I missed this. I have removed this line.

Reply:
And about the merging `next` into my branch, would that be part of this issue? If so, is there any documentation I can refer to, so that I could highlight the breaking changes as per the standards?